Plugins/Geant4/src/Geant4Converters.cpp

```diff
@@ -38,6 +41,7 @@
 #include "G4ThreeVector.hh"
 #include "G4Transform3D.hh"
 #include "G4Trd.hh"
+#include "G4Trap.hh"
```

Comment:
Please remove.

Reply:
Why should I remove #include "G4Trap.hh"

Reply:
Sorry that was on a different line, G4Trapp has to stay of course.